ppr-ui/src/composables/useRegistration.ts

```diff
@@ -42,21 +42,12 @@ export const useRegistration = (setSort: RegistrationSortIF) => {
     }
   }
 
-  const getStatusDescription = (status: APIStatusTypes | MhApiStatusTypes): string => {
+  const getStatusDescription = (status: APIStatusTypes | MhApiStatusTypes,
+    isChild: boolean, isPpr: boolean): string => {
```

Comment:
`isPpr` used only in tests? If yes then we need a default value, otherwise, it will be undefined on like 50.

Reply:
isPpr is passed in to the table row component, and I forward it to getStatusDescription from the tableRow, however I can add a default value if you believe it will be better!

![image](https://github.com/bcgov/ppr/assets/77707952/32bbe5e2-91f4-48af-a134-439ade76a919)

Reply:
Sounds good, we should be ok then, thanks for clarifying.